### ISSUE TITLE
✨[feature Request]: Dark Mode/ Light Mode Toggle

### ISSUE THREAD

**Issue by Poushmita** [Anjaliavv51/Retro#65]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I want to add a `dark mode/light mode toggle`.

### Use Case

Allow users to switch between **dark mode and light mode** to improve visual comfort and match the UI to their preferences or environmental conditions.

### Benefits

Dark mode can `reduce eye strain` and `provide better readability, especially` in low-light conditions. Light mode, on the other hand, is `better suited for bright environments, providing contrast and clarity`.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24EXTD contributor
- [X] I have starred the repository

**Comment by Anjaliavv51** [comment]
@Poushmita go ahead make sure feature is implemented to all html pages in website, then only level 3 will be assigned to u.

**Comment by Shalinis19137** [comment]
I would like to work on issue #65 in the feature Request : Dark Mode/Light Mode Toggle repository. Could you please assign it to me? Thank you!

**Comment by Anjaliavv51** [comment]
> I would like to work on issue #65 in the feature Request : Dark Mode/Light Mode Toggle repository. Could you please assign it to me? Thank you!

No u can't work as it is already assigned to @Poushmita

**Comment by Poushmita** [comment]
@Shalinis19137   If you want then you can group with me for this issue.

**Comment by Poushmita** [comment]
@Anjaliavv51  okay .

**Comment by Anjaliavv51** [comment]
> @Shalinis19137 If you want then you can group with me for this issue.

Yes go ahead of both don't have any issue please

**Comment by Shalinis19137** [comment]
Yes,I want to group with you.Its my pleasure mam

**Comment by Poushmita** [comment]
@Anjaliavv51  Okay mam.

**Comment by Anjaliavv51** [comment]
@Poushmita don't address me as mam , I think we PPL have similar age.

**Comment by Poushmita** [comment]
@Anjaliavv51  oh! okay .

**Comment by Poushmita** [comment]
# After Applying Dark Mode
## Screenshots
<img width="959" alt="c1" src="https://github.com/user-attachments/assets/9d2e284d-5f81-4a9f-86ce-019f669c42a3">

<img width="959" alt="c2" src="https://github.com/user-attachments/assets/eaa7ae30-665c-4b15-9592-2ee95f9f2212">

**The text colour after changing mode will be adjusted.**



@Anjaliavv51  kindly response me, that is it okay or I would `change `some `background colour( texts also accordingly) ` of some sections  for `Darkmode `?
- If it is okay then, I will just apply this for other pages and adjust some text colours.
- If you want more then I will have to make many changes accordingly.
`I have no problem with any of the above options , please reply what you want?`

**Comment by Anjaliavv51** [comment]
@Poushmita no website background color is pink change that related background to black

**Comment by Poushmita** [comment]
@Anjaliavv51 you want `pink` backgrounds `to be black` in `dark mode`.. 
Okay, then I am working on this.

**Comment by Anjaliavv51** [comment]
@Poushmita @Shalinis19137 any progress?

**Comment by Poushmita** [comment]
@Anjaliavv51 yes I have done almost with the code. It is just giving some problem due to the previously  set values. I am working on it.
